### PR TITLE
register fluid kerenls to phi [part 7]

### PR DESCRIPTION
**Pull request by huangjiyi**
### PR types

Others

### PR changes

Others

### Describe

Pcard-67000

Register fluid kernels of 23 ops to phi：
- nccl
- nce
- nop
- number_count
- pad2d
- pad_constant_like
- partial_allgather
- partial_concat
- partial_recv
- partial_send
- partial_sum
- polygon_box_transform
- positive_negative_pair
- precision_recall
- proximal_adagrad
- proximal_gd
- prroi_pool
- prune_gate_by_capacity
- pull_box_extended_sparse
- pull_box_sparse
- pull_gpups_sparse
- pull_sparse
- pull_sparse_v2

### CODE REVIEW

File: paddle/fluid/operators/partial_sum_op.cu [hunk 1]
```diff
@@ -233,18 +233,3 @@ class PartialSumGradOpCUDAKernel : public framework::OpKernel<T> {
 
 }  // namespace operators
 }  // namespace paddle
-
-namespace ops = paddle::operators;
-REGISTER_OP_CUDA_KERNEL(partial_sum,
```
Comment (YuanRisheng) [inline paddle/fluid/operators/partial_sum_op.cu:238]: 可以后续pr补一下注释，说明一下这里删除的原因

**Review by YuanRisheng — APPROVED** [review]

File: paddle/fluid/operators/partial_sum_op.cu (hunk 1 above)
Comment (huangjiyi) [reply paddle/fluid/operators/partial_sum_op.cu:238]: 好的

**Review by Aurelius84 — APPROVED** [review]
LGTM for data type registeration

**Review by zhwesky2010 — APPROVED** [review]
LGTM for change unity_build_rule

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**